AndroidPubSub/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.4.0'
```

Comment:
Has this been tested?
Will it work in older versions of Android Studio?

Reply:
If you could find the minimum Android studio version that would work with this version of Gradle Plugin, please add it to the `README`.

Reply:
Yes, this has been tested to work on the latest version. As for compatibility with the older version, the compatibility is listed in https://developer.android.com/studio/releases/gradle-plugin.